pandas/core/groupby/groupby.py

```diff
@@ -2737,10 +2737,11 @@ def cummin(self, axis=0, **kwargs):
         -------
         Series or DataFrame
         """
+        skipna = kwargs.get("skipna", True)
         if axis != 0:
             return self.apply(lambda x: np.minimum.accumulate(x, axis))
 
-        return self._cython_transform("cummin", numeric_only=False)
+        return self._cython_transform("cummin", numeric_only=False, skipna=skipna)
```

Comment:
Ideally `cummin/max` could handle `kwargs` the same way as `cumsum` does, but the problem is the forced `numeric_only=False`. 

As a followup change, it would be nice to remove this inconsistency (especially since `numeric_only=True` would just be ignored). Would this require a deprecation cycle, or could current behavior of ignoring `numeric_only=True` be considered a bug? 

Making this change could enable actually exposing `skipna` and `numeric_only` in the signatures of these cumulative functions, instead of having docs that just show `*args`, `**kwargs`

Reply:
> or could current behavior of ignoring numeric_only=True be considered a bug?

I think so, yes.

> Making this change could enable actually exposing skipna and numeric_only in the signatures of these cumulative functions, instead of having docs that just show *args, **kwargs

Yah, looks like we currently silently ignore args/kwargs, which isnt great.

Reply:
Ok sounds good, will plan to leave as followon since should be an independent improvement from these changes